redisvl/extensions/llmcache/semantic.py

```diff
@@ -23,7 +25,6 @@ class SemanticCache(BaseLLMCache):
     entry_id_field_name: str = "entry_id"
```

Comment:
We should use the same pattern to define constant field names, beyond just the vector field.

Reply:
I'd like to pull out the changing of class fields to constants into a new PR. Should be a quick one.

Reply:
I'd like to move the changing of all these class params into a constants file to another PR. Should be a quick one.